tests/integration/deprecations/comma-separated-selector-test.js

```diff
@@ -0,0 +1,55 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+import PageObject from 'dummy/tests/page-object';
+
+// In the old ember testing API we have a `render` method available
+// only in the `moduleForComponent` test helper. Unfortunatelly it requires
+// some component name to be passed as a first argument. That's why we pass
+// a `calculating-device` here despite the fact that we don't actually need it.
+moduleForComponent('calculating-device', 'Integration | comma separated selectors', {
+  integration: true
+});
+
+test('usage of comma-separated selector in the scope leads to a deprecation', function(assert) {
```

Comment:
Should we also be asserting that the deprecation does *not* show up when the selector does *not* contain a comma?

Reply:
Thanks, good point.